lib/views/admin/complete_profile/complete_profile.dart

```diff
@@ -259,6 +260,7 @@ class _AdminCompleteProfileState extends State<AdminCompleteProfile> {
                             try {
                               db.createProfile(
                                   restaurantNameController.text,
+                                    restaurantLocationController.text,
```

Comment:
Where are you convert text editing controllers into geo points? Location type is not text